applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaSpatialGridContainer.h

```diff
@@ -34,7 +34,7 @@
 #ifndef SOFA_GPU_CUDA_CUDASPATIALGRIDCONTAINER_H
 #define SOFA_GPU_CUDA_CUDASPATIALGRIDCONTAINER_H
 
-#include <SofaSphFluid/SpatialGridContainer.h>
+#include <SofaSphFluid/src/SofaSphFluid/SpatialGridContainer.h>
```

Comment:
Needing to do this is not normal.

To use SofaSphFluid headers, you must have `find_package(SofaSphFluid)` and `target_link_library(${PROJECT_NAME} PUBLIC SofaSphFluid)` in your CMakeLists.

If it does not work, the problem comes from SofaSphFluid.

Reply:
Here is a full example of how to add a soft dependency to SofaSphFluid : https://github.com/sofa-framework/sofa/search?q=SofaSphFluid+OpenCL